### DISCUSSION TITLE
Connect Security Onion with Azure Sentinel

### DISCUSSION THREAD

**Discussion by MJRH54** [Security-Onion-Solutions/securityonion#5568] [Ideas] [answered]
Hello, currently I tried to connect security onion to azure sentinel for logs monitoring in the cloud.

Unfortunately I could not do it.

Someone know how to make this process? 

My references are:

https://github.com/Azure/Azure-Sentinel/tree/master/DataConnectors/microsoft-logstash-output-azure-loganalytics
https://docs.securityonion.net/en/2.3/logstash.html#logstash
https://docs.securityonion.net/en/16.04/syslog-output.html

**Comment by dougburks** [comment, marked as answer]
https://docs.securityonion.net/en/16.04/syslog-output.html is for the old Security Onion 16.04 which is past End Of Life.

https://docs.securityonion.net/en/2.3/syslog-output.html is the version of the page for the current Security Onion 2.3.

**Reply by MJRH54** [reply]
Thanks for reply, the solution was forward all syslog to another server :)
